docs/integrations/data-integrations/couchbase.mdx

```diff
@@ -20,14 +20,16 @@ Before proceeding, ensure the following prerequisites are met:
 This handler is implemented using the `couchbase` library, the Python driver for Couchbase.
 
 The required arguments to establish a connection are as follows:
-* `host`: the host name or IP address of the Couchbase server
+* `connection_string`: the Connection string to specify the cluster endpoint
 * `bucket`: the bucket name to use when connecting with the Couchbase server
 * `user`: the user to authenticate with the Couchbase server
 * `password`: the password to authenticate the user with the Couchbase server
 * `scope`:  scopes are a level of data organization within a bucket. If omitted, will default to `_default`
 
+Note: The connection string expects the couchbases:// or couchbase:// part.
+
 <Tip>
-If you are using Couchbase Capella, the `host` should be the connection string for the cluster. When you navigate to your cluster, the connection string can be found under the Connect tab.
+If you are using Couchbase Capella, the `connection_string` can be found out when you navigate under the Connect tab.
```

Comment:
Let's make this:
If you are using Couchbase Capella, you can find the connection_string under the Connect tab.

Please make the same change in the README.